toolbox/load_agol_layer/src/load_agol_layer.js

```diff
@@ -105,13 +105,15 @@ const main = async ({ layer: layerName, save }) => {
    * an upsert mutation is performed. We simply take the entire field array excluding
    * the primary key
    */
-
   const isUpsert = layerConfig.upsert;
 
   const updateFields = layerConfig.fields
     .filter((field) => !field.isPrimaryKey)
     .map((field) => field.outputName);
 
+  // add geometry to updated fields
+  updateFields.push("geometry");
```

Comment:
before this changes, we were upserting location polygons without their geometries 🤦

Reply:
dang sorry for missing this in review! just so i understand, this was hard to catch because there were already geometries in these layers - they just weren't updated, right?